style/style.scss

```diff
@@ -0,0 +1,70 @@
+$color-title:#FF0000;
+$color-subtitle:#B22222;
+$color-name:#A52A2A;
+$background:#6495ED;
+$fontbase:20px ;
+$font-family:"Times New Roman", Times, serif;
+$images:270px;
+$font-title:35px;
+$font-intro:25px;
+$font-subtitle:30px;
+$font-name:28px;
```

Comment:
Are you using these variables multiple times? cause if not, it's not necessary to put a variable for it.